tests/integration/test_vulnerability_detector/test_cpe_helper/data/configuration_template/configuration_cpe_indexing_wrong_values.yaml

```diff
@@ -0,0 +1,75 @@
+- sections:
+    - section: vulnerability-detector
+      elements:
+        - enabled:
+            value: 'yes'
+        - run_on_start:
+            value: 'yes'
+        - provider:
+            attributes:
+              - name: redhat
+            elements:
+              - enabled:
+                  value: 'no'
+        - provider:
+            attributes:
+              - name: canonical
+            elements:
+              - enabled:
+                  value: 'no'
+        - provider:
+            attributes:
+              - name: debian
+            elements:
+              - enabled:
+                  value: 'no'
+        - provider:
+            attributes:
+              - name: msu
+            elements:
+              - enabled:
+                  value: 'yes'
+              - update_interval:
+                  value: 1h
+        - provider:
+            attributes:
+              - name: alas
+            elements:
+              - enabled:
+                  value: 'no'
+        - provider:
+            attributes:
+              - name: arch
+            elements:
+              - enabled:
+                  value: 'no'
+        - provider:
+            attributes:
+              - name: nvd
+            elements:
+              - enabled:
+                  value: 'yes'
+              - path:
+                  value: NVD_JSON_PATH
+
+    - section: sca
+      elements:
+        - enabled:
+            value: 'no'
+
+    - section: rootcheck
+      elements:
+        - disabled:
+            value: 'yes'
+
+    - section: syscheck
+      elements:
+        - disabled:
+            value: 'yes'
+
+    - section: wodle
+      attributes:
+        - name: syscollector
+      elements:
+        - disabled:
+            value: 'yes'
```

Comment:
This should be the default for VD IT tests.
Accepted until the development of the new framework is over.

Reply:
Ok